cli/internal/plugins/bzr/bzr.go

```diff
@@ -12,41 +12,70 @@ import (
 	"github.com/gobuffalo/buffalo-cli/plugins/plugprint"
 )
 
-type Buffalo struct{}
+const (
+	name        = "bzr"
```

Comment:
Try to avoid package level state and context. What if a second bzr plugin is added?